devtools/config-doc-maven-plugin/src/main/java/io/quarkus/maven/config/doc/generator/MarkdownFormatter.java

```diff
@@ -38,4 +38,9 @@ protected String tooltip(String value, String javadocDescription) {
         // we don't have tooltip support in Markdown
         return "`" + value + "`";
     }
+
+    @Override
+    protected String escapeDefaultValue(String defaultValue) {
+        return "```" + defaultValue + "```";
```

Comment:
No line breaks?

Reply:
Between the values, it's a `, ` separator, mostly because the column is tight and it doesn't look that bad.